src/ar.hrl

```diff
@@ -19,9 +19,7 @@
 	]).
 
 -define(DEFAULT_RESPONSE_HEADERS,
-	[
-		{<<"Access-Control-Allow-Origin">>, <<"*">>}
-	]).
+	#{<<"access-control-allow-origin">> => <<"*">>}).
```

Comment:
HTTP headers are case-insensitive but it feels safer to preserve the case for the response headers. What would you say?

Reply:
This is actually cowboy specific behaviour. Cowboy explicitly asks the developer to always refer to headers using only lowercase letters.

Reply:
Does cowboy explain this? I think, for us it's just a matter of backwards-compatibility, in case some clients rely on headers specified in the particular case.

Reply:
Yep, super explicitly:

> The header name must be given as a lowercase binary string. While header names are case insensitive, Cowboy requires them to be given as lowercase to function properly.

https://ninenines.eu/docs/en/cowboy/2.6/manual/cowboy_req.set_resp_headers/

Reply:
Ouch, that's fucked up. Okay, then we have no choice.